launchable/commands/record/build.py

```diff
@@ -183,6 +191,17 @@ def build(ctx: click.core.Context, build_name: str, source: List[str], max_days:
     uniq_submodules = {commit_hash: (name, repo_dist, commit_hash)
                        for name, repo_dist, commit_hash, in sources + submodules}.values()
```

Comment:
[question]
Do you have a specific reason you are making dict and then dict_values?
Can you initialize as list without going through dict, for readability?

[nits]
make this list instead of dict_values.
```python
uniq_submodules = list({commit_hash: (name, repo_dist, commit_hash)
                       for name, repo_dist, commit_hash, in sources + submodules}.values())
```

Reply:
> Do you have a specific reason you are making dict and then dict_values?

To be honest, I'm not original author of it, so, I don't know the reason.

> Can you initialize as list without going through dict, for readability?

In my opinion, if we passed the test and don't change the request payload format, we can change it.